src/Superdesk/ContentApiSdk/API/Authentication/AuthenticationInterface.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * This file is part of the PHP SDK library for the Superdesk Content API.
+ *
+ * Copyright 2015 Sourcefabric z.u. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2015 Sourcefabric z.ú.
+ * @license http://www.superdesk.org/license
+ */
+
+namespace Superdesk\ContentApiSdk\API\Authentication;
+
+use Superdesk\ContentApiSdk\Client\ClientInterface;
+use Superdesk\ContentApiSdk\Exception\AuthenticationException;
+
+interface AuthenticationInterface
+{
+    /**
+     * Instantiate the object.
+     *
+     * @param ClientInterface $client
+     */
+    public function __construct(ClientInterface $client);
```

Comment:
Well, technically it's not required to implement the Abstract class, then again you might use your own client solution when implementing the interface (imo not a good idea). So by putting it here, it would force developers implementing other classes to keep re-using already available classes.